docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+version: "3.4"
+services:
+  docker:
+    image: homework25:latest
+    container_name: docker-compose
+    build: /home/authz/git
+    ports:
+      - 80:80
```

Comment:
Everything was installed just fine. But when i connect to the host and do a
```
$ curl localhost
curl: (56) Recv failure: Connection reset by peer
```
as you see there is an error :thinking:

Reply:
checked, found that flask works by default on 5000 port